packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -52,13 +52,7 @@ export function useGridEditRows(apiRef: GridApiRef) {
     if (params.cellMode === 'view') {
       return;
     }
-
-    const cellCommitParams = apiRef.current.getEditCellPropsParams(params.id, params.field);
-    if (!cellCommitParams.props.error) {
-      // We commit the change when there is no error
```

Comment:
how are you handling this now? error was a sort of predefined prop

Reply:
`commitCellChange` now checks if there's an error. If there's no error, it publishes `GRID_CELL_EDIT_PROPS_CHANGE_COMMITTED`. The handler of this event is who updates the row.